docs/topics/mta-cli-args.adoc

```diff
@@ -0,0 +1,210 @@
+|--userIgnorePath |Specify a location, in addition to `${user.home}/.{LC_PSN}/ignore/`, for {ProductShortName} to identify files that should be ignored.
+|--userLabelsDirectory |Specify a location for {ProductShortName} to look for custom Target Runtime Labels. The value can be a directory containing label files or a single label file. The Target Runtime Label files must use the [x-]`.windup.label.xml` suffix. The shipped Target Runtime Labels are defined within `${ProductShortName}_HOME/rules/migration-core/core.windup.label.xml`.
+|--userRulesDirectory |Specify a location, in addition to `<{ProductShortName}_HOME>/rules/` and `${user.home}/.{LC_PSN}/rules/`, for {ProductShortName} to look for custom {ProductShortName} rules. The value can be a directory containing ruleset files or a single ruleset file. The ruleset files must use the [x-]`.windup.xml` suffix.
+|--version |Display the {ProductShortName} version.
+|--skipSourceCodeReports |This option skips generating a _Source code report_ when generating the application analysis report. Use this advanced option when concerned about source code information appearing in the application analysis.
+|====
+
+[id="cli-input-argument_{context}"]
+== Specifying the input
+
+A space-delimited list of the path to the file or directory containing one or more applications to be analyzed. This argument is required.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--input <INPUT_ARCHIVE_OR_DIRECTORY> [...]
+----
+
+[id="cli-input-file-type-arguments_{context}"]
+
+Depending on whether the input file type provided to the `--input` argument is a file or directory, it will be evaluated as follows depending on the additional arguments provided.
+
+Directory::
++
+[cols="1,1,1",options="header"]
+|====
+| --explodedApp
+| --sourceMode
+| Neither Argument
+
+| The directory is evaluated as a single application.
+| The directory is evaluated as a single application.
+| Each subdirectory is evaluated as an application.
+|====
+
+File::
++
+[cols="1,1,1",options="header"]
+|====
+| --explodedApp
+| --sourceMode
+| Neither Argument
+
+| Argument is ignored; the file is evaluated as a single application.
+| The file is evaluated as a compressed project.
+| The file is evaluated as a single application.
+|====
+
+[id="cli-output-argument_{context}"]
+== Specifying the output directory
+
+Specify the path to the directory to output the report information generated by {ProductShortName}.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--output <OUTPUT_REPORT_DIRECTORY>
+----
+
+* If omitted, the report will be generated in an `<INPUT_ARCHIVE_OR_DIRECTORY>.report` directory.
+* If the output directory exists, you will be prompted with the following (with a default of N).
++
+[source,options="nowrap",subs="attributes+"]
+----
+Overwrite all contents of "/home/username/<OUTPUT_REPORT_DIRECTORY>" (anything already in the directory will be deleted)? [y,N]
```

Comment:
This should be changed.

Now if the directory exists and the `--overwrite` argument has not been provided, the command will throw an error:

`Error: output dir //home/username/<OUTPUT_REPORT_DIRECTORY> already exists and --overwrite not set`

Reply:
Amended